src/pages/index.html

```diff
@@ -4,16 +4,380 @@
+      <h5 >  Resource Management</h5>
+      <p >Resource management is the process of pre-planning, 
+        scheduling, and allocating
+         your resources to maximize efficiency.</p>
+    </div> <div class="box">
+      <img src="../assets/images/Layer 2.svg" alt="">
+      <h5 >  Resource Management</h5>
+      <p >Resource management is the process of pre-planning, 
+        scheduling, and allocating
+         your resources to maximize efficiency.</p>
+    </div> <div class="box">
+      <img src="../assets/images/Layer 3.svg" alt="">
+      <h5 >  Resource Management</h5>
+      <p >Resource management is the process of pre-planning, 
+        scheduling, and allocating
+         your resources to maximize efficiency.</p>
+    </div> <div class="box">
+      <img src="../assets/images/Layer 4.svg" alt="">
+      <h5 >  Resource Management</h5>
+      <p >Resource management is the process of pre-planning, 
+        scheduling, and allocating
+         your resources to maximize efficiency.</p>
+    </div> <div class="box">
+      <img src="../assets/images/Layer5.svg" alt="">
+      <h5 >  Resource Management</h5>
+      <p >Resource management is the process of pre-planning, 
+        scheduling, and allocating
+         your resources to maximize efficiency.</p>
+    </div> <div class="box">
+      <img src="../assets/images/Layer 6.svg" alt="">
+      <h5 >  Resource Management</h5>
+      <p >Resource management is the process of pre-planning, 
+        scheduling, and allocating
+         your resources to maximize efficiency.</p>
+    </div>
+  </div>
+
+</div>
+</div>
+
+<div class="customers ">
+
+  <div class="make">
+    <h3> See what our customers are saying  </h3>
+    <p> 2,157 people have said how good we are  </p>
+  </div>
+   
+
+<div class="people ">
+
+
+
+
+<div class="card1" >
+
+<img src="../assets/images/BG (1).png" alt="">
+    <p >“I can’t tell how easy it was to grow my small shop with it”</p>
+    <h5>Jenny Wilson</h5>
+<span> Founder, ExtendSales </span>
+  </div>
+
+  <div class="card1" >
+
+
+    <img src="../assets/images/BG.png" alt="">
+    <p >“I can’t tell how easy it was to grow my small shop with it”</p>
+      <h5>Jenny Wilson</h5>
+  <span> Founder, ExtendSales </span>
+    </div>
+
+</div>
+</div>
+<div class="decision ">
+  <div class="make">
+    <h3>Make the wise decision for your business  </h3>
+    <p> Choose from our affordable 3 packages  </p>
+  </div>
+ 
+<div class="business ">
+  <div class="card1" >
+
+    <div class="Premium  d-flex flex-column">
+      <span> Basic </span>
+  
+      <span> $99 </span>
+    </div>
+
+  
+      <p >Best for medium business owners, startups who needes landing page for their business.</p>
+      <a class="Started" href="#"> Get Started  </a>
+      <ul>
+    <li class="active"> What’s included:</li>
+    <li > <a href="#"> 160+ UI Blocks</a> </li>
+    <li > <a href="#"> Best for Developers, Freelancers</a> </li>
+    
+  </ul>
+    </div>
+
```

Comment:
![Capture](https://github.com/shaban011147/landing/assets/106445705/f4b981f1-95c9-495a-b11d-d8c38c33de08)

1- not the same font family in the design
2- Why didn't you? When you press the card, the button changes color

Reply:
done